### DISCUSSION TITLE
Getting this fatal error while push

### DISCUSSION THREAD

**Discussion by saroj-17** [Orchid-DOTNET/BscCSIT-2023#16] [General]
![Screenshot from 2023-03-24 19-37-31](https://user-images.githubusercontent.com/115420182/227539783-057a84be-4dc4-4244-8aa6-97d4aa20f875.png)

**Comment by enigma037** [comment]
@sarojiam17 - can you go to directory BScCSIT-2023? and call git command from there? Not from the SarojBhandari Folder.
Also I have noticed you have not pull new changes from EmmaFacultyBranch. 
Let me delete your branch, but save your changes before I delete your branch. Ping me once you re done.

**Reply by saroj-17** [reply]
Ok it's showing same thing while trying from BSCCSIT too it is also asking for authentication  and fatal error so please delete my branch so that i can create new one an I think it will work
